content/learn/text-editing.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Text-Editing Workflow
+weight: 6800
+type: page
+abstract:  "Our process for adding publication content and data into Quire via text editor"
+online: false
+---
+
+{{< q-checklist >}}
+
+- [ ] Open project in text editor
+- [ ] Delete starter content
+- [ ] Complete the `publication.yml`
+- [ ] Use publication TOC to structure placeholder Markdown files and landing pages
```

Comment:
Should we link some of these items to relevant areas in the docs? Like this could go to: https://quire.getty.edu/documentation/getting-started/#create-a-publication-outline